microsite/data/plugins/pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Pulumi
+author: pulumi.com
+authorUrl: https://www.pulumi.com
+category: Infrastructure
+description: View Pulumi stack information in Backstage.
+documentation: https://github.com/pulumi/pulumi-backstage-plugin
+iconUrl: https://www.pulumi.com/logos/brand/avatar-on-white.png
+npmPackageName: '@pulumi/backstage-plugin-pulumi'
```

Comment:
Thanks for adding this - I wasn't able to find the package on npm yet. Is this the intended package name where it's going to be published?

Reply:
@tudi2d yes, working on that at this very moment. 😄 

Should I move the `addedDate` forward?

Reply:
no, that's perfectly fine!! :)